src/commonMain/kotlin/exchange.dydx.abacus/state/model/TradingStateMachine.kt

```diff
@@ -1264,7 +1293,7 @@ open class TradingStateMachine(
                 val modifiedFills = fills?.toIMutableMap() ?: mutableMapOf()
                 var subaccountFills = fills?.get(subaccountText)
                 if (staticTyping) {
-                    val newFills = internalState.wallet.account.subaccounts?.get(subaccountNumber)?.fills?.toIList()
+                    val newFills = internalState.wallet.account.subaccounts[subaccountNumber]?.fills?.toIList()
```

Comment:
can we safely assume this if subaccount orders state is hidden behind a feature flag?

Reply:
oh, this is behind the FF. sorry!